getting-started/create-a-mobile-property.md

```diff
@@ -10,11 +10,11 @@ Before you can set up your mobile property, complete the following prerequisites
 
 Launch is fully integrated with your Adobe ID. User permissions are managed through the Admin Console with other Adobe products and solutions from the Creative Cloud, Document Cloud, and Experience Cloud.
 
-For detailed instructions on how to create groups and add users for Launch, see [User permissions](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html).
+For detailed instructions on how to create groups and add users for Launch, see [user permissions](https://docs.adobe.com/content/help/en/launch/using/reference/admin/user-permissions.html).
 
 ### Log in to Experience Platform Launch
 
-After Experience Platform Launch rights have been added to your Adobe ID, log in to Launch. You can do this by going to [https://launch.adobe.com](https://launch.adobe.com) or by logging in to the Experience Cloud \([https://experiencecloud.adobe.com](https://experiencecloud.adobe.com)\), navigating to the Activation page, and clicking on Launch.
+After Experience Platform Launch rights have been added to your Adobe ID, log in to Launch. You can do this by going to [https://launch.adobe.com](https://launch.adobe.com) or by logging in to the Experience Cloud ([https://experiencecloud.adobe.com](https://experiencecloud.adobe.com)), navigating to the Activation page, and selecting Launch.
```

Comment:
should `https://launch.adobe.com` be updated to `https://experience.adobe.com/#/data-collection` ?